recipe/meta.yaml

```
@@ -9,7 +9,7 @@ source:
   sha256: 4a0373e84316cf4bb711779a258a200f778daf3e6cfc51cae8a5df383e3383dd
 
 build:
-  number: 0
+  number: 1
   noarch: python
```

Is this what you had in mind Matt?

```suggestion
  noarch: python
  string: unix_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [unix]
  string: win_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}   # [win]
```

---

Also just to clarify the hashes are different on their own (taken [from CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=864825&view=results ))

* UNIX: `conda-smithy-3.30.4-pyh707e725_1.conda`
* Windows: `conda-smithy-3.30.4-pyh7428d3b_1.conda`

---

@jakirkham, can you apply this suggestion? I don't have permissions.

---

Yep added

Was unsure whether we wanted to include this